src/parse/asp/objects.go

```diff
@@ -363,7 +364,7 @@ func (l pyList) Operator(operator Operator, operand pyObject) pyObject {
 			}
 			panic("Cannot add list and " + operand.Type())
 		}
-		return append(l, l2...)
+		return slices.Clip(append(l, l2...))
```

Comment:
Do we need to apply this change to the `pyFrozenList` case a few lines above too?